managers/examens/Examen.java

```diff
@@ -0,0 +1,12 @@
+package managers.examens;
+
+import java.time.Duration;
+import java.util.Date;
+
+public abstract class Examen {
```

Comment:
- [ ] TODO: changer le nom?
- [ ] TODO: changer le nom pour toutes les classes managers en suivant ce nom:
```suggestion
public abstract class ExamenManager {
```
_Mettre Manager après le nom de chaque "métier" permet de mieux reconnaître la classe en plus de sa localisation dans le package managers. **Et on se rapproche d'un nom plus conventionnel**_